docs/knowledgebase/getting-started/index.md

```diff
@@ -77,20 +77,18 @@ runtimes. You will need to configure your Rust compiler to use
 [`nightly` builds](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) to allow you to
 compile Substrate runtime code to the Wasm target. There is
 [a GitHub Issue](https://github.com/paritytech/substrate/issues/1252) that describes this
-requirement in greater detail.
+requirement in detail.
 
 #### Rust Nightly Toolchain
 
-Developers that are building _with_ Substrate (as opposed to the developers building Substrate
-_itself_) should use a specific Rust nightly version that is known to be compatible with the version
-of Substrate they are using. For instance, the Polkadot client specifies this information in its
-[release notes](https://github.com/paritytech/polkadot/releases). The Substrate Developer Hub Node
-Template uses an
+Developers building _with_ Substrate should use a specific Rust nightly version that is known to
+be compatible with the version of Substrate they are using. Refer to the Polkadot [release notes](https://github.com/paritytech/polkadot/releases) for more specific information. The Substrate Node Template uses an
 [init script](https://github.com/substrate-developer-hub/substrate-node-template/blob/master/scripts/init.sh)
 and [Makefile](https://github.com/substrate-developer-hub/substrate-node-template/blob/master/Makefile)
-to specify the Rust nightly version and encapsulate the following steps. Use Rustup to install the
+to specify the Rust nightly version by encapsulating the following steps. Use Rustup to install the
```

Comment:
I don't think the changes in terminology improve the clarity or accuracy of these paragraphs.